Example/swiftframe.config

```diff
@@ -1,5 +1,5 @@
 {
-    "backgroundColor": "#5787E4",
```

Comment:
Might be worth checking if there's an easy way to parse YAML in Swift, since it's a little easier to read for configuration files.